src/components/dumb/OeSelect.vue

```diff
@@ -11,17 +11,39 @@
     <div class="vl-select__inner">
       <div class="vl-input-field">
         <div
+          v-if="selectedOption"
           class="vl-select__item vl-select__item--selectable"
           :data-id="selectOptionLabel"
           :data-value="selectedOption"
         >
           {{ selectOptionLabel }}
         </div>
+        <div
+          v-else
+          class="vl-select__item vl-select__item--selectable item--placeholder"
+          :data-id="selectOptionLabel"
+          :data-value="selectedOption"
+        >
+          {{ $props.placeholder }}
+        </div>
       </div>
     </div>
     <div v-if="showResults" class="vl-select__list vl-select__list--dropdown">
       <div class="vl-select__list" dir="ltr" role="listbox">
         <template v-for="(option, index) in options" :key="'select-' + index">
+          <div
```

Comment:
Dit kan wat korter, die `data` attributes hebben we niet nodig denk ik, dan kan de placeholder ook buiten de `v-for`:

```vue
    <div v-if="showResults" class="vl-select__list vl-select__list--dropdown">
      <div class="vl-select__list" dir="ltr" role="listbox">
        <div class="vl-select__item vl-select__item--choice is-placeholder">
          {{ $props.placeholder }}
        </div>
        <template v-for="(option, index) in options" :key="'select-' + index">
          <div
            class="vl-select__item vl-select__item--choice vl-select__item--selectable"
            :class="{ 'is-highlighted': isEqual(selectedOption, option), '': !isEqual(selectedOption, option) }"
            @click="selectOption(option)"
          >
            {{ props.customLabel(option) }}
          </div>
        </template>
      </div>
    </div>
```